lib/internal/abort_controller.js

```diff
@@ -35,6 +37,11 @@ function customInspect(self, obj, depth, options) {
 }
 
 class AbortSignal extends EventTarget {
+  constructor() {
+    // eslint-disable-next-line no-restricted-syntax
```

Comment:
Out of curiosity, why is this comment needed?

Reply:
> Out of curiosity, why is this comment needed?

It's because of this: https://github.com/nodejs/node/blob/c7d2a4544be23dc6f199c79eabfa03ea7691dc6f/lib/.eslintrc.yaml#L22-L23.